runtimes/clone/src/lib.rs

```diff
@@ -697,10 +697,11 @@ impl_runtime_apis! {
 
 		fn execute_block(block: Block, state_root_check: bool, sig_check: bool, select: frame_try_runtime::TryStateSelect) -> Weight {
 			log::info!(
-				target: "runtime::clone", "try-runtime: executing block #{} ({:?}) / root checks: {:?} / sanity-checks: {:?}",
+				target: "runtime::spiritnet", "try-runtime: executing block #{} ({:?}) / root checks: {:?} / sig check: {:?} / sanity-checks: {:?}",
```

Comment:
This is the clone runtime, so clone is the right name 😄

Reply:
Thanks, stupid copy-paste error.